packages/react-composites/src/composites/CallWithChatComposite/components/AddPeopleDropdown.tsx

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import {
+  DefaultButton,
+  IButtonStyles,
+  IContextualMenuItem,
+  IContextualMenuProps,
+  IContextualMenuStyles,
+  PrimaryButton,
+  Stack,
+  useTheme
+} from '@fluentui/react';
+import { _DrawerMenu, _DrawerMenuItemProps } from '@internal/react-components';
+import copy from 'copy-to-clipboard';
+import React, { useMemo, useState } from 'react';
+import { CallWithChatCompositeIcon } from '../../common/icons';
+import { copyLinkButtonContainerStyles, copyLinkButtonStackStyles } from '../../common/styles/PeoplePaneContent.styles';
+import { convertContextualMenuItemToDrawerMenuItem } from '../ConvertContextualMenuItemToDrawerMenuItem';
+import { drawerContainerStyles } from '../styles/CallWithChatCompositeStyles';
+import { iconStyles, themedCopyLinkButtonStyles, themedMenuStyle } from './AddPeopleDropdown.styles';
+import { PreparedDialpad, PreparedDialpadStrings } from './PreparedDialpad';
+
+/** @private */
+export interface AddPeopleDropdownStrings extends PreparedDialpadStrings {
+  copyInviteLinkButtonLabel: string;
+  openDialpadButtonLabel: string;
+  peoplePaneAddPeopleButtonLabel: string;
+}
+
+/** @private */
+export interface AddPeopleDropdownProps {
+  inviteLink?: string;
+  mobileView?: boolean;
+  strings: AddPeopleDropdownStrings;
+}
+
+/** @private */
+export const AddPeopleDropdown = (props: AddPeopleDropdownProps): JSX.Element => {
+  const theme = useTheme();
+
+  const { inviteLink, strings, mobileView } = props;
+  const [showDialpad, setShowDialpad] = useState(false);
+
+  const menuStyleThemed: Partial<IContextualMenuStyles> = useMemo(() => themedMenuStyle(theme), [theme]);
```

Comment:
nitty nit: You should not need to explicitly specify the type here (`useMemo` should figure it out).